app/javascript/articles/__tests__/__snapshots__/Article.test.jsx.snap

```diff
@@ -237,15 +237,6 @@ Object {
                   >
                     1 min read
                   </small>
-                  <button
```

Comment:
(blocking) I think the `Article` component test should be updated to:

- reverse the change to these snapshots
- add a test that covers the `saveable={false}` case